hack/generated/config/manager/manager.yaml

```diff
@@ -25,6 +25,7 @@ spec:
       containers:
       - args:
         - --enable-leader-election
+        - --v=1
```

Comment:
Should we set this higher?

Reply:
While we're in Alpha, yes - we kind of expect issues. Change it back to 1 when we hit Beta, maybe?

Reply:
I've put this up to 2 for now. That means we're not logging full API requests/responses (which we get at 3) but we're logging everything else.

Avoiding full requests/responses (except in our test logs for now) seems prudent given they could theoretically contain secrets at the moment since we don't do any pruning AND we don't put secrets into their own Kubernetes secrets object. We'll need to do something about that - I've got #1762 filed tracking that.